package.json

```diff
@@ -81,6 +80,7 @@
     "rollup-plugin-json": "^4.0.0",
     "rollup-plugin-peer-deps-external": "^2.2.4",
     "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-typescript2": "^0.36.0",
```

Comment:
Reason for using this package?

Reply:
@rollup/plugin-typescript supports only one outDir

Reply:
We didn't plan to change the plugin for typescript here. The current plugin is considered more stable and faster

Reply:
with current plugin we have to duplicate the same build with different outDir and than build time will be two times greater

Reply:
seems like it was fixed in 11 version, I will just update @rollup/plugin-typescript